creation/interfaces-and-networks.md

```diff
@@ -733,3 +733,103 @@ spec:
 
 > **Note:** for some NICs (e.g. Mellanox), the kernel module needs to be
 > installed in the guest VM.
+
+### macvtap
+
+In `macvtap` mode, virtual machines are directly exposed to an host networking
```

Comment:
a host.

I think that the first part of this sentence is redundant. Can't we just say that VMs are connected directly to the L2 ...?